### ISSUE TITLE
React.render is deprecated

### ISSUE THREAD

**Issue by infowaybs** [thereactivestack/meteor-react-router-ssr#13]
After updating the React package, I'm getting this message:

```
warning.js:45 Warning: React.render is deprecated. Please use ReactDOM.render from require('react-dom') instead.
```

I think React.render is used on this package. Right?

**Comment by eXon** [comment]
The package haven't been updated for React 0.14 yet

**Comment by infowaybs** [comment]
Do you have any plans to update it? Should I open another issue about updating it to React 0.14.1?

**Comment by eXon** [comment]
It will be updated very soon.

**Comment by eXon** [comment]
Should be fixed in 0.3.0!
